### PR TITLE
Install letter_opener gem

### PR DESCRIPTION
**Pull request by benlinton**
We'll need to smoke screen test this gem with delayed jobs.  I imagine we shouldn't run into any problems, but let's try it out before merging.  I'll test when time permits.

https://github.com/ryanb/letter_opener

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = false
```
Comment (jhanggi) [inline config/environments/development.rb]: I was playing around with it. It seemed to only work for me if you changed `perform_deliveries = true`. Did you get it working?

Comment (jhanggi) [inline config/environments/development.rb]: @benlinton ^^^

Comment (benlinton) [inline config/environments/development.rb]: I cracked the gem open.  It appears we will have to call `.deliver` for the gem to work.  And I also had to set `perform_deliveries = true` in `config/environments/development.rb`.

Comment (jhanggi) [inline config/environments/development.rb]: It seemed to work for me when I set `perform_deliveries = true` in `development.rb` and if I kicked off delayed_job (`script/delayed_job run`). If it works for you, I'm cool with trying this out for a little while.

Comment (benlinton) [inline config/environments/development.rb]: Firing up delayed job worked for me as well.  Awesome.  I updated `config/environments/development.rb`.  This PR should be ready to go.

**Pull request merged**